### DISCUSSION TITLE
Watching is incredibly dangerous and poorly explained to new users

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#10869] [Projects and Issues]
I regularly file issues and PRs to the [github/docs](http://github.com/github/docs) repository, and periodically, someone will reply (by email) to a notification for an update to one of these items.

I'm reminded of [Bedlam DL3](https://en.wikipedia.org/wiki/Email_storm#Examples).

GitHub makes it *very* easy to watch a repository without making it remotely obvious the amount of daily/weekly activity (and notifications) one could expect to receive by virtue of taking this action.

It should be much harder to subscribe to a firehose 🔥⛲ than it is today.

---

By default when users are added to a new organization, it seems that the default is to 
> (Automatically watch repositories) When you’re given push access to a repository, automatically receive notifications for it.

![image](https://user-images.githubusercontent.com/2119212/152009415-34dec836-5082-4b24-8d75-b31d6ab6e026.png)

This feature is documented:

https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/viewing-your-subscriptions#diagnosing-why-you-receive-too-many-notifications

But, for any nontrivially sized organization, this setting is a recipe for disaster.

---

I'd like to suggest that:
* clicking watch should show a note with indications about the average weekly/daily notification flow
* automatically watch repositories for new organizations should be turned off
  * it could be replaced by a view listing each repository's weekly/daily flow and an option to subscribe when they accept the invitation

**Comment by hsandt** [comment]
Also see https://github.com/github/feedback/discussions/12504 for reducing notifications load via batching (the weekly/daily notification flow idea could be a good combination with that). Although ultimately, we'll want to prevent unwanted subscriptions as you said.
